Sources/ConsoleKit/Command/Command/Inputs.swift

```diff
@@ -0,0 +1,36 @@
+/// The structure of the inputs that a command can take
+///
+///     struct Signature: Intpus:
+///         let name = Argument<String>(name: "name")
+///     }
+public protocol Inputs {
```

Comment:
Maybe we call this `AnySignature` to match everything else?

Reply:
Ah, I see now that you have to write this a lot. 

```swift
struct Signature: Inputs {
    ...
}
```

I wonder if there's any way we could get around this... Could we provide the required extensions on the command instead of on the inputs? If we store the signature statically, then we don't need `Input`'s protocol requirement. That let's us do:

```swift
struct CowSay: Command {
    struct Signature {
        let eyes = Option<String>(name: "eyes", short: "e")
    }
    static let signature = Signature()
}
```

Then we should be able to create a mirror with `Mirror(reflecting: Self.signature)`

Reply:
Assuming we do that, what type should [`AnyCommandRunnable.inputs`](https://github.com/vapor/console/pull/92/files#diff-453d0a7ee566e188bf560196e8f28e3fR6) be?

Reply:
Hmm, `Any` I guess?

Reply:
Good, but if we remove that, we can't automatically create a `Signature` instance, which is really nice for the end user. Should we still do that?

We could keep the `Input` protocol and move the `arguments` and `options` properties.